nemo_skills/pipeline/verl/ppo.py

```diff
@@ -229,6 +229,11 @@ def ppo_verl(
     server_args: str = typer.Option("", help="Any extra arguments to pass to the server"),
     wandb_project: str = typer.Option("nemo-skills", help="Weights & Biases project name"),
     disable_wandb: bool = typer.Option(False, help="Disable wandb logging"),
+    final_ckpt_path: str = typer.Option(None, help="Where to put the final checkpoint"),
+    save_last_ckpt: bool = typer.Option(
```

Comment:
```suggestion
    convert_last_ckpt_to_hf: bool = typer.Option(
```
that feels more descriptive? That last checkpoint is always saved, but here we are saving it in hf format